konnector.js

```diff
@@ -1,254 +1,160 @@
 'use strict'
 
-const request = require('request')
 const cheerio = require('cheerio')
 const moment = require('moment')
 
 const {
     log,
-    baseKonnector,
-    filterExisting,
-    linkBankOperation,
-    saveDataAndFile,
-    models
+    BaseKonnector,
+    saveBills
 } = require('cozy-konnector-libs')
-const Bill = models.bill
-
-module.exports = baseKonnector.createNew({
-  name: 'Numéricable',
-  description: 'konnector description numericable',
-  vendorLink: 'https://www.numericable.fr/',
-
-  category: 'isp',
-  color: {
-    hex: '#53BB0F',
-    css: '#53BB0F'
-  },
-
-  dataType: ['bill'],
-
-  models: [Bill],
-
-  fetchOperations: [
-    login,
-    parsePage,
-    customFilterExisting,
-    customSaveDataAndFile,
-    customLinkBankOperation
-  ]
+
+const request = require('request-promise')
+
+const rq = request.defaults({
```

Comment:
If you do : 
```javascript
const rq = request({
  cheerio: true,
  jar: true,
  json: false
})
```

You have `followAllRedirects: true` for free and don't need resolveWithFullResponse since you already get a wrapped body as promise result like [here](https://github.com/cozy/cozy-konnector-malakoffmederic/blob/master/index.js#L63) for example. And you do not need to wrap the body with cheerio yourself each time.

Reply:
I am not a big fan of having directly a cheerio object as promise result, especillay when it's called `$`. It feel really magical to me and not enough explicit. Maybe we could discuss and improve this point in future version. Let's go with it for now. 👍